.github/workflows/bench-browser.yml

```diff
@@ -0,0 +1,27 @@
+name: Run Benchmarks In The Browser
+on:
+  # manual trigger
```

Comment:
@themighty1 To test this, I'd also add `pull_request:` here. Once we see it successfully runs on GH we can remove `pull_request:` again.

Reply:
Thanks, Im testing it in another repo https://github.com/themighty1/tlsn/actions